pkg/minikube/notify/notify.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notify
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/blang/semver"
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+	"k8s.io/minikube/pkg/minikube/constants"
+	"k8s.io/minikube/pkg/version"
+)
+
+const updateLinkPrefix = "https://github.com/kubernetes/minikube/releases/tag/v"
+
+var timeLayout = time.RFC1123
+
+var lastUpdateCheckFilePath = constants.Minipath + "/last_update_check"
+
+func GetUpdateText() (string, error) {
```

Comment:
It might make your code easier if this function just never returns an error, and returns empty strings in all error cases. (You can log the error though).